pyccl/tracers.py

```diff
@@ -292,6 +298,9 @@ class NumberCountsTracer(Tracer):
     def __init__(self, cosmo, has_rsd, dndz, bias, mag_bias=None):
         self._trc = []
 
+        # we need the distance functions at the C layer
+        cosmo.compute_distances()
+
```

Comment:
Do you also need to initialize growth if you need RSDs? Maybe it happens automatically.

Reply:
Where is the growth function called? 

Note that calling the growth function in python does the init but it does not happen in C.

Reply:
Line 321, but you've already answered this, so all good